frontend/src/Components/Navbar.tsx

```diff
@@ -0,0 +1,158 @@
+            <a href="#" className="block px-4 py-1 text-gray-700 hover:bg-gray-200">Feedback</a>
+          </MobileDropdown>
+        </li>
+        <li>
+          <a href="#" className="block px-4 py-2 text-gray-700 hover:bg-gray-200">
+            Help
+          </a>
+        </li>
+      </ul>
+    </div>
+  );
+};
+
+const Navbar = () => {
+  const [isHovering, setIsHovering] = useState<Record<DropdownName, boolean>>({
+    dashboard: false,
+    settings: false,
+  });
+
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  const handleMouseEnter = (dropdown: DropdownName) => {
+    setIsHovering({ ...isHovering, [dropdown]: true });
+  };
+
+  const handleMouseLeave = (dropdown: DropdownName) => {
+    setIsHovering({ ...isHovering, [dropdown]: false });
+  };
+
+  const handleMenuToggle = () => {
+    setIsMenuOpen(!isMenuOpen);
+  };
+
+  return (
+    <div className="mt-2 w-full flex justify-between items-center rounded-[4px] p-4 shadow-lg">
+      <div className="ml-4 text-blue-500 font-bold">
+        <span className="bg-blue-500 py-1 px-2 text-white rounded-full mr-1">T</span>
+        TASK
+      </div>
+      <div className="flex items-center bg-black/5 h-8 w-full rounded-3xl px-8 gap-2 mx-8">
+        <div>
+          <FiSearch />
+        </div>
+        <input type="text" placeholder="search anything" className="w-full bg-transparent outline-none" />
+      </div>
+      <div className="mr-4 hidden md:block">
+        <ul className="flex gap-8">
+          <li
+            className="text-blue-500 font-medium relative"
+            onMouseEnter={() => handleMouseEnter('dashboard')}
+            onMouseLeave={() => handleMouseLeave('dashboard')}
+          >
+            <a href="#" className="flex items-center">
+              Dashboard
+            </a>
+            <div
+              className={`absolute top-full left-0 bg-white shadow-md text-sm ${isHovering.dashboard ? 'block' : 'hidden'}`}
+            >
+              <a href="#" className="block px-4 py-1 text-gray-700 hover:bg-gray-200">Profile</a>
+              <a href="#" className="block px-4 py-1 text-gray-700 hover:bg-gray-200">Logout</a>
+              <a href="#" className="block px-4 py-1 text-gray-700 hover:bg-gray-200">Security</a>
+              <a href="#" className="block px-4 py-1 text-gray-700 hover:bg-gray-200">Notification update</a>
+              <a href="#" className="block px-4 py-1 text-gray-700 hover:bg-gray-200">Language Prefrences</a>
+            </div>
+          </li>
+          <li
+            className="text-blue-500 font-medium relative"
+            onMouseEnter={() => handleMouseEnter('settings')}
+            onMouseLeave={() => handleMouseLeave('settings')}
+          >
+            <a href="#" className="flex items-center">
+              Settings
+            </a>
+            <div
+              className={`absolute top-full left-0 bg-white shadow-md text-sm ${isHovering.settings ? 'block' : 'hidden'}`}
```

Comment:
why you are needed to a hovering state any reasons

Reply:
Beacause for the dropdown menu.